debian/copyright

```diff
@@ -0,0 +1,38 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: argobots
+Source: <url://example.com>
+
+Files: *
+Copyright: <years> <put author's name and email here>
+           <years> <likewise for another author>
+License: <special license>
+ <Put the license of the package here indented by 1 space>
+ <This follows the format of Description: lines in control file>
+ .
+ <Including paragraphs>
+
+# If you want to use GPL v2 or later for the /debian/* files use 
+# the following clauses, or change it to suit. Delete these two lines
+Files: debian/*
+Copyright: 2019 John E. Malmberg <jemalmbe@rial-10vm9.rial.hpdd.intel.com>
```

Comment:
Can we not get a better e-mail address in here?

Reply:
Yes.  That was auto-generated because I did not have the DEB* environment variables set, and I missed fixing that.